pylivetrader/misc/parallel_utils.py

```diff
@@ -37,7 +37,10 @@ def wrapper(args_list):
             for task in concurrent.futures.as_completed(tasks):
                 args = tasks[task]
                 task_result = task.result()
-                result[args] = task_result
+                if isinstance(args, list):
+                    result[str(args)] = task_result
```

Comment:
this doesn't make much sense... turning a list to str. what are you trying to achieve?